src/extension.ts

```diff
@@ -1,5 +1,6 @@
 'use strict';
 import * as vscode from 'vscode';
+let moment = require('moment');
```

Comment:
Why let?

Reply:
Using `const` would have also worked, and in hindsight might have been marginally better. But it would not be a significant enough improvement in my mind to warrant reopening the branch and changing it.

Reply:
Sure, just asked why. Always try to keep definitions clean.
No problem.